tools/distpartitioning/data_shuffle.py

```diff
@@ -12,82 +12,88 @@
 from datetime import timedelta
 from dataset_utils import get_dataset
 from utils import read_partitions_file, read_json, get_node_types, \
-                    augment_node_data, augment_edge_data, get_ntypes_map, \
+                    augment_edge_data, get_ntypes_map, \
                     write_dgl_objects, write_metadata_json
 from gloo_wrapper import alltoall_cpu_object_lst, alltoallv_cpu, \
                     alltoall_cpu, allgather_sizes, gather_metadata_json
 from globalids import assign_shuffle_global_nids_nodes, \
                     assign_shuffle_global_nids_edges, get_shuffle_global_nids_edges
 from convert_partition import create_dgl_object, create_metadata_json, validateDGLObjects
```

Comment:
`validateDGLObjects` is not in `convert_partition.py` and it's not being used anywhere.

Reply:
"validateDGLObjects" function is removed. This was added only for debugging purposes and should never be released to the general users of this library. Removed this function name from the imports section of this file.